test/test_var_view.py

```diff
@@ -25,3 +27,5 @@ def test_get_stringifier():
 
             s = strifier(value)
             assert isinstance(s, text_type)
+
+test_get_stringifier()
```

Comment:
This should be under a `if __name__ == '__main__'` block so that pytest can execute it on its own without it being executed at import time.